src/backends/interflop-mca/interflop_mca.c

```diff
@@ -281,6 +327,34 @@ static void _set_mca_seed(const bool choose_seed, const uint64_t seed) {
   _set_seed_default(&random_state, choose_seed, seed);
 }
 
+/* Set the mca seed for the simple generators */
+static void _set_mca_seed_simple(const bool choose_seed,
+                                 const unsigned int seed) {
+  _set_seed_simple(&random_state_simple, choose_seed, seed);
+}
+
+/* Set the random number generator to be used */
+static void _set_mca_rng_mode(const mca_rng_mode mode) {
+  if (mode >= mca_rng_mode_drand) {
+    logger_error("--%s invalid value provided, must be one of: "
+                 "{mt, rand, random, drand}.",
+                 key_rng_mode_str);
+  }
+  MCALIB_RNG_MODE = mode;
+}
+
+/* Get a new identifier for the calling thread */
```

Comment:
Could you please expand the comment to indicate why we need our own gettid implementation when fixing the seed. 

(syscall(gettid) does not give deterministic ids and thus does not allow reproducibility for predictable thread programs).